### PR TITLE
foreman_openscap 0.4.2

### PR DESCRIPTION
**Pull request by shlomizadok**
Includes relaxing of deface

### CODE REVIEW

File: rubygem-foreman_openscap/rubygem-foreman_openscap.spec [hunk 1]
```diff
@@ -27,7 +27,7 @@ Requires: foreman >= 1.8.0
 
 Requires: scap-security-guide
 Requires: %{?scl_prefix}rubygem(deface)
```
Comment (domcleal) [inline rubygem-foreman_openscap/rubygem-foreman_openscap.spec]: Add `< 2.0` here to match the gemspec.

File: rubygem-foreman_openscap/rubygem-foreman_openscap.spec [hunk 2]
```diff
@@ -27,7 +27,7 @@ Requires: foreman >= 1.8.0
...
-Requires: %{?scl_prefix}rubygem(scaptimony) >= 0.3.0
+Requires: %{?scl_prefix}rubygem(scaptimony) >= 0.3.2
```
Comment (domcleal) [inline rubygem-foreman_openscap/rubygem-foreman_openscap.spec]: also mirror this in the BuildRequires below

**Comment by shlomizadok** [comment]
Changed. Thanks

**Comment by domcleal** [comment]
Which repos is this update intended for?  1.8 has 0.4.0, 1.9 and nightly have 0.4.1.

**Comment by shlomizadok** [comment]
Time to unify? should work on 1.8 - nightly (since adding [add_smart_proxy_filters](https://github.com/theforeman/foreman/commit/c3b335360965c123887b9df31ba1a6cd204fcb50#diff-35ac9743013bfaf29815fb6e0b40ef1fR8)

**Comment by domcleal** [comment]
Sure, will do all three then.  Thanks @shlomizadok, building.

**Pull request merged**